test/example/deployment.yaml

```diff
@@ -30,3 +44,6 @@ spec:
               sleep 10 &
               wait  # interrupted quickly on signal.
           done
```

Comment:
This code works with nginx image? It may work if the container include a shell,
but I think this means that nginx default entrypoint will not run, so nothing
will listen to port 80.

Reply:
Updated the new code set, without nginx image